core/deployment/src/main/java/io/quarkus/deployment/dev/DevModeConfig.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.deployment.dev;
+
+import java.util.List;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "devmode")
+public class DevModeConfig {
```

Comment:
For me the idea is to extend `io.quarkus.deployment.LiveReloadConfig` and move the top class comment on top of the existing properties.

I would include your property first to have something clean.

Reply:
> I would include your property first to have something clean.

You mean leave it `additional-watched-resources`?

Reply:
Yes, the property name is OK IMO. I was just saying you should put yours before the dummy ones we already have in `io.quarkus.deployment.LiveReloadConfig`.

Reply:
Done.